package-rfcx/rfcx/audio.py

```diff
@@ -108,3 +119,103 @@ def downloadGuardianAudio(token, destination_path, guardian_id, min_date, max_da
         print("Finish download on {}".format(guardian_id))
     else:
         print("No data found on {} - {} at {}".format(start[:-10], end[:-10], guardian_id))
+
+# v2
+def save_audio_file2(token, destination_path, audio_id, source_audio_extension='opus'):
+    """ Prepare `url` and `local_path` and save it using function `__save_file` 
+        Args:
+            destination_path: Audio save path.
+            audio_id: RFCx audio id in format `{stream_id}_t{start time}.{end time}_g{audio gain}_f{audio format}`
+            source_audio_extension: (optional, default= '.opus') Extension for saving audio files.
+
+        Returns:
+            None.
+
+        Raises:
+            TypeError: if missing required arguements.
+    
+    """
+    url = "https://api.rfcx.org/internal/assets/streams/" + audio_id + "." + source_audio_extension
```

Comment:
There is a more efficient url for downloading files: https://media-api.rfcx.org/internal/assets/streams/

Reply:
I don't think it is `audio_id`... isn't it stream_id?

Reply:
`audio_id` in here is in shape `{stream_id}_t{start time}.{end time}_g{audio gain}_f{audio format}`. I think it is new version of audio id on api.